javascript/react/fundamentals/the-react-component.md

```diff
@@ -27,58 +27,77 @@ aspects:
 
 ---
 
-# The **React** component
+# The React component
 
 ---
 ## Content
 
-In **React** a `component` represents a fundamental *class* you can work with to build re-usable and independent `UI` blocks.
+In **React**, a component represents a fundamental *class* you can work with to build re-usable and independent `UI` blocks.
```

Comment:
Maybe instead of "class you can work with to build" you can say "unit of logic"?